helm/lisk-core/README.md

```diff
@@ -0,0 +1,42 @@
+# Helm chart for deploy a lisk-core statefulset
+
+This is the initial version of this chart, and it's still consided experimental.
+
+## Requirements
+
+- Helm installed on the local computer
+- A kubernetes cluster with helm tiller deployed
+- Support for persistent storage
+
+## Installation
+
+Inside chart main directory - where `Chart.yaml` is located:
+
+### Install dependencies
+
+`helm dependency build`
```

Comment:
Is this really necessary?
```
~> helm dependency build
No requirements found in ./charts.
```

Reply:
my bad. it was necessary in the initial version I did, when postgresql was running in a different pod from lisk core. will remove it.